website/.spelling

```diff
@@ -360,6 +360,7 @@ hashcode
 hashtable
 high-QPS
 historicals
+histogramBuckets
```

Comment:
I don't think this should be needed since this word occurs within code blocks in the docs.

Reply:
Makes sense, pushing changes to the following commit.